app/presenters/tree_builder_action.rb

```diff
@@ -7,7 +7,7 @@ def tree_init_options(_tree_name)
 
   # level 0 - root
   def root_options
-    [t = N_("All Actions"), t]
+    [t = _("All Actions"), t]
```

Comment:
@chessbyte the initial conversions used `N_`, instead of `_` in TreeBuilder, but we've since settled on `_`.